server/chart_config.json

```diff
@@ -894,7 +907,7 @@
     ],
     "isOverview": true,
     "relatedChart": {
-      "title": "Marital Status Per Capita",
+      "title": "Marital Status Rate",
```

Comment:
citizenship and other pop breakdowns are currently listed as "per capita" so this was meant to be consistent with those.

Reply:
Find it a bit confusing if we should 20% together with "per capita", maybe choose one of the two, I feel rate is closer to what we use daily?

Reply:
i figure for demographics related stats, per capita is more accurate..

Reply:
let's keep rates:
http://www.oecd.org/social/family/SF_3_1_Marriage_and_divorce_rates.pdf

Reply:
Done. Thanks!